ios/engine/KMEI/KeymanEngine/Classes/Settings/SettingsViewController.swift

```diff
@@ -229,19 +290,30 @@ open class SettingsViewController: UITableViewController {
   }
   
   private func performAction(for indexPath: IndexPath) {
+    let cellIdentifier = itemsArray[indexPath.row]["reuseid"]
```

Comment:
Does this need to be inside the `switch indexPath.section { case 0:` block? Even if not currently a problem, could it be a problem in the future if we added additional sections?

Reply:
It's supposed to _replace_ that section.  Should be `switch cellIdentifier { case "<identifier>":` instead.

Reply:
It doesn't currently replace that switch though?

Reply:
... Oh wait, you were right.  Forgot about the 'section' vs 'row' distinction.